apps/docs/js-interpreter.md

```diff
@@ -0,0 +1,167 @@
+# Upgrading the JS Interpreter
+
+## Component Overview
+
+Before embarking on an upgrade of the JS interpreter,
+you need to know what all the different little pieces
+are and how they relate to each other.
+
+### Neil Fraser's JS-Interpreter Repository
+
+[NeilFraser/JS-Intepreter](https://github.com/NeilFraser/JS-Interpreter)
+is the main repository where development of the interpreter takes place. It gets updated
```

Comment:
I don't know who they are or how to contact them, but maybe Neil would be willing to give us that info?